app/controllers/support_interface/support_users_controller.rb

```diff
@@ -1,7 +1,8 @@
 module SupportInterface
   class SupportUsersController < SupportInterfaceController
     def index
-      @support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
+      @unsorted_support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
```

Comment:
```suggestion
      unsorted_support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
```

does this need to be an instance variable? Do we use it in the index controller, or do we always use `@support_users`?

Reply:
Nope you are right, I was supposed to do this but forgot, thank you. 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/9963/commits/aff0cfda3b33d05565c5d083e6476d378c307650